.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk:
```

Comment:
You might want to check the project settings to see if you allow the new git squash PR feature.  https://github.com/blog/2141-squash-your-commits